### ISSUE TITLE
X marks the Close on modals

### ISSUE THREAD

**Issue by Alexander-Mjoberg** [careerfairsystems/arkad-app#325]
Our QR designer made a nifty suggestion that instead of having close on "Need an account" and "Help", we could use a (X) button in the top right. Then it would also be alot closer to to the help button and the card would be less long. Here's a rough idea
![image](https://user-images.githubusercontent.com/15044398/66186855-a97a9c80-e683-11e9-83ed-16d282c769df.png)

**Comment by arpi09** [comment]
Good idea! Should put it on the card top right corner, make the x blue and no background colour.
